js/app.js

```diff
@@ -30,9 +30,9 @@ var DataView = Backbone.View.extend({
     });
     this.$el.html(html);
     this.view = this._makeMultiView(this.dataset, this.$el.find('.multiview'));
-    this.dataset.query({
+    /*this.dataset.query({
       size: this.dataset.recordCount
-    });
+    });*/
```

Comment:
It would be good to add a comment on this or just delete. Future contributors will wonder why this is commented out.